### ISSUE TITLE
Contact contains square "Close" button in FF

### ISSUE THREAD

**Issue by svlasenko** [dhanur/Sunible-Beta#45]
Steps to reproduce:
1. Open Sunible page in FireFox browser
2. Click on "Contact"
3. Observe Contact window 

Actual result: Close button looks like a square but should be circular.

![image](https://f.cloud.github.com/assets/1586612/755243/066b7bce-e5a7-11e2-929f-cfdf81f8de03.png)

**Comment by rlytvynchuk** [comment]
Cannot reproduce. returned to QA

Tested on FF20(mac), FF22(win, mac)
![scr_contact_us](https://f.cloud.github.com/assets/1098812/757941/3cf7d71e-e700-11e2-9172-ee3e90757422.PNG)
![srn_ff22_mac](https://f.cloud.github.com/assets/1098812/757942/56f0f3bc-e700-11e2-9b7d-832443ba33e3.png)

**Comment by svlasenko** [comment]
I still can reproduce on my local server.

**Comment by dhanur** [comment]
Good catch, Sveta. I don't see this is as a major usability or design issue and I'm also not able to reproduce. Also, Firefox traffic is quite minimal per Google Analytics. Shall we close?

**Comment by svlasenko** [comment]
Maybe Roman can look on this one and find out something.

**Comment by rlytvynchuk** [comment]
I cannot do this at home, 'cause issue is not reproducible on Virtual machine Windows. But I reproduced it on my regular job. So it can be fixed only there. Since issue is minor and not in current sprint let's get back to it in when it's planned for some sprint

**Comment by dhanur** [comment]
Yes, thanks. Our focus is on agreed sprint items + emergency issues as needed.
